src/main/java/uk/gov/justice/digital/service/DomainService.java

```diff
@@ -56,7 +57,7 @@ protected void handleError(final Exception e) {
         final StringWriter sw = new StringWriter();
         final PrintWriter pw = new PrintWriter(sw);
         e.printStackTrace(pw);
-        System.err.print(sw.getBuffer().toString());
+        logger.error(sw.getBuffer().toString());
```

Comment:
You don't need to writers or this method. Replace all usages of this method with an appropriate call to `logger.error("Some message", theException)`. We should *not* be outputting anything to STDOUT.

Reply:
done